src/use-mutation/use-mutation.ts

```diff
@@ -0,0 +1,56 @@
+import { StimulusUse, StimulusUseOptions } from '../stimulus_use'
+import { method } from '../support/index'
+import { MutationController } from './mutation-controller'
+
+export interface MutationControllerOptions {
+  element?: Element
+}
+
+export interface MutationOptions extends MutationObserverInit, MutationControllerOptions, StimulusUseOptions {
+}
+
+export class UseMutation extends StimulusUse {
+  controller: MutationController
+  observer: MutationObserver
+  targetElement: Element
+  options: MutationOptions
+
+  constructor(controller: MutationController, options: MutationOptions = {}) {
+    super(controller, options)
+
+    this.targetElement = options?.element || controller.element
+    this.controller = controller
+    this.options = options
+    this.observer = new MutationObserver(this.mutation)
+
+    this.enhanceController()
+    this.observe()
+  }
+
+  observe = () => {
+    this.observer.observe(this.targetElement, this.options)
```

Comment:
maybe we could wrap it in a try/catch and report the error to Stimulus ErrorHandler

```suggestion
   try {
      this.observer.observe(this.targetElement, this.options)
    } catch (error) {
      this.controller.application.handleError(error,"At a minimum, one of childList, attributes, and/or characterData must be true",{})
    }
```

Reply:
Sounds entirely reasonable!